recipes/kibana/.dockerignore

```diff
@@ -0,0 +1 @@
+Dockerfile
```

Comment:
If the repository is moved to Quay.io on day, then Dockerfile must be part of the Docker *build context* and thus should not be ignored.